bin/ramalama

```diff
@@ -1,21 +1,51 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
-import argcomplete
+import glob
 import os
 import errno
 import subprocess
 import sys
 
+def get_pipx_dir():
+    python_version = f'{sys.version_info.major}.{sys.version_info.minor}'
+    argcompletion_path = os.path.expanduser(f'~/.local/pipx/venvs/argcomplete/lib/python{python_version}/site-packages')
```

Comment:
Should we just add
I think this is wrong and would only find argcomplete, and not other packages installed by pipx? Should we just add .local/pipx/venvs to sys.paths?

Reply:
argcomplete is our only dependency, we call CLI versions of omlmd and huggingface so the python imports directory is irrelevant

Reply:
We'd have to loop though and add every:

~/.local/pipx/venvs/*/lib/python{python_version}/site-packages'

dir, which might be worth it

Reply:
Added a much more generic version